packages/fx-core/resource/package.nls.json

```diff
@@ -232,6 +235,8 @@
   "error.ProjectEnvNotExistError": "Environment %s not found. Make sure the config.%s.json file exist.",
   "error.InvalidEnvNameError": "Environment name can only contain letters, digits, _ and -.",
   "error.UpgradeCanceledError": "If you don't want to upgrade your project, please install another version of Teams Toolkit (version <= 2.10.0).",
+  "error.ConsolidateCanceledError": "If you don't want to upgrade your project, please install another version of Teams Toolkit (version <= ????).",
+  "error.OperationNotSupportedForExistingAppError": "Operation is not supported for existing app: %s.",
```

Comment:
Looks the error message is not used and can be removed?

Reply:
My bad, add import to ```ConsolidateCanceledError```.